tests/unit/ext/switches/mock_switch.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2016 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+
+
+from haas import config, deferred, model
+from haas.ext.obm.ipmi import Ipmi
+from haas.model import db
+from haas.test_common import config_testsuite, config_merge, \
+                             fresh_database, fail_on_log_warnings
+
+from sqlalchemy import Column, Integer, ForeignKey, String
```

Comment:
You don't need to (and shouldn't) import these from the sqlalchemy module; they're re-exported by `db`, so you can just do:

```python

foo = db.Column(db.Integer, ...)
bar = db.Column(...)
```

And so forth. Have a look at what the other database objects do.